api/src/main/resources/ConfigurationEntry.hbm.xml

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://hibernate.sourceforge.net/hibernate-mapping-3.0.dtd">
+<hibernate-mapping>
+	<class name="org.openmrs.module.patientmatching.ConfigurationEntry" table="patientmatching_configurationEntries">

```

Comment:
Ah, so you didn't come up with this table name; rather, you are inheriting it from earlier authors of the module. This table belongs to the patient matching module (our convention is to prepend table names with module ID to avoid conflicts), so it would be possible to rename the table with a liquibase changeset. When I saw the hibernate mapping file, I assumed you were creating the table. If the table already exists for the module, then we can address changing the table name to follow conventions at a later time.